random-beans/src/test/java/io/github/benas/randombeans/beans/WildCardCollectionBean.java

```diff
@@ -0,0 +1,550 @@
+/*
+ * The MIT License
+ *
+ *   Copyright (c) 2016, Mahmoud Ben Hassine (mahmoud.benhassine@icloud.com)
+ *
+ *   Permission is hereby granted, free of charge, to any ? extends Runnable obtaining a copy
```

Comment:
Hello Pascal, 
That's a funny generic version of the MIT license :smile:  You must be `Runnable` to obtain a copy of the software.. Run baby run! I'll fix that, don't worry..
Cheers